moninag/static/src/ts/app/servers/model.ts

```diff
@@ -1,16 +1,12 @@
 
 export class Server {
-    constructor(
-        public id: number,
-        public name: string,
-        public address: string,
-        public state: string,
-        public userid: number
-    ) { }
-}
+    public id: number;
+    public name: string;
+    public address: string;
+    public state: string;
+    public userid: number;
 
-export const states = [
-                        'NotSelected',
-                        'Production',
-                        'Staging'
-                      ];
+    constructor() {
+        this.state = 'NotSelected';
```

Comment:
Probably 'NotSelected' will be changed to 'Disabled' in the nearest future (since the change already in pull request)